src/dev/performance/run_performance_cli.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { run } from '@kbn/dev-cli-runner';
+import { REPO_ROOT } from '@kbn/utils';
+import Fsp from 'fs/promises';
+import path from 'path';
+
+function setupEnv() {
+  process.env.TEST_ES_URL = 'http://elastic:changeme@localhost:9200';
+  process.env.TEST_ES_DISABLE_STARTUP = 'true';
+}
+
+function cleanupEnv() {
+  delete process.env.ELASTIC_APM_TRANSACTION_SAMPLE_RATE;
+  delete process.env.ELASTIC_APM_SERVER_URL;
+  delete process.env.ELASTIC_APM_SECRET_TOKEN;
+  delete process.env.ELASTIC_APM_ACTIVE;
+  delete process.env.ELASTIC_APM_CONTEXT_PROPAGATION_ONLY;
+  delete process.env.ELASTIC_APM_ACTIVE;
+  delete process.env.ELASTIC_APM_SERVER_URL;
+  delete process.env.ELASTIC_APM_SECRET_TOKEN;
+  delete process.env.ELASTIC_APM_GLOBAL_LABELS;
+  delete process.env.TEST_ES_URL;
+  delete process.env.TEST_ES_DISABLE_STARTUP;
+  delete process.env.TEST_PERFORMANCE_PHASE;
```

Comment:
FYI, rather than mutating the global process.env var we could also pass an `env` arg to `procRunner` that forces the value of these env vars to `undefined`.

Reply:
To which one of the procRunners?

Reply:
It would need to be passed to all of the procRunners. The `env` option of `procRunners.run()` defaults to `process.env`.